bndtools.core/src/bndtools/views/repository/RepositoriesView.java

```diff
@@ -464,6 +466,17 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
                 return Status.OK_STATUS;
             }
         }.schedule();
+        KeyAdapter adapter = new KeyAdapter() {
```

Comment:
Here is a Eclipsepedia artifact about this. 
https://wiki.eclipse.org/FAQ_How_do_I_hook_into_global_actions,_such_as_Copy_and_Delete%3F

So you would want to do something like:
```
IActionBars actionBars= getViewSite().getActionBars();
   actionBars.setGlobalActionHandler(
      ActionFactory.REFRESH.getId(),
      refreshAction);	
```